server/admin/server.go

```diff
@@ -94,16 +94,10 @@ func UseLogger(logger slog.Logger) {
 	log = logger
 }
 
-// filesExists reports whether the named file or directory exists.
-func fileExists(name string) bool {
-	_, err := os.Stat(name)
-	return !os.IsNotExist(err)
-}
-
 // NewServer is the constructor for a new Server.
 func NewServer(cfg *SrvConfig) (*Server, error) {
 	// Find the key pair.
-	if !fileExists(cfg.Key) || !fileExists(cfg.Cert) {
```

Comment:
To be perfectly honest, it's this kind of thing that I would keep any day to avoid further tightening the coupling between packages. `dex` is about the lowest level, so OK, but please do not assume it's always best to add an import just remove a few lines of boilerplate. OK to keep these changes, but next time save this for a PR to discuss.

Reply:
Alright, I understand minor duplication > importing in some scenarios and will keep this in mind.

Reply:
Thanks. In this case it's really NBD, but in the very near future we are going to break the repo into at least two different modules, and that could get hairy.